src/app/shared/services/http.service.ts

```diff
@@ -31,8 +32,8 @@ export class HttpService {
     this.toastService.showSuccess(message);
   }
 
-  getViews(): Observable<any> {
-    return this.http.get('api/testtool/views').pipe(catchError(this.handleError()));
+  getViews(): Observable<{ [key: string]: View }> {
+    return this.http.get<{ [key: string]: View }>('api/testtool/views').pipe(catchError(this.handleError()));
```

Comment:
Hi, thanks for the PR. 

Could you change `{ [key: string]: View }` to `Record<string, View>` that reads a bit better?

Reply:
Or Map<string, View>?. Wouldn't that be better, Sergi?

Reply:
Map does more than what @Banshaan needs it for in this case. `Record` is a Utility type: https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Changed { [key: string]: View } to Record<string, View>